delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/directives/ResponseToJson.scala

```diff
@@ -2,56 +2,28 @@ package ch.epfl.bluebrain.nexus.delta.sdk.directives
 
 import akka.http.scaladsl.model.MediaTypes.`application/json`
 import akka.http.scaladsl.model.StatusCodes.OK
-import akka.http.scaladsl.model.{ContentType, HttpCharsets, MediaType}
-import akka.http.scaladsl.server.Directives._
-import akka.http.scaladsl.server.Route
 import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.RemoteContextResolution
 import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.encoder.JsonLdEncoder
 import ch.epfl.bluebrain.nexus.delta.rdf.utils.JsonKeyOrdering
-import ch.epfl.bluebrain.nexus.delta.sdk.directives.DeltaDirectives.{requestMediaType, unacceptedMediaTypeRejection}
-import ch.epfl.bluebrain.nexus.delta.sdk.directives.Response.{Complete, Reject}
+import ch.epfl.bluebrain.nexus.delta.sdk.directives.Response.Complete
 import ch.epfl.bluebrain.nexus.delta.sdk.directives.ResponseToJson.UseRight
 import ch.epfl.bluebrain.nexus.delta.sdk.marshalling.HttpResponseFields
 import ch.epfl.bluebrain.nexus.delta.sdk.marshalling.RdfMarshalling._
-import ch.epfl.bluebrain.nexus.delta.sdk.syntax._
 import io.circe.Json
 import monix.bio.{IO, UIO}
 import monix.execution.Scheduler
 
-sealed trait ResponseToJson {
-  def apply(): Route
-}
-
 object ResponseToJson extends JsonValueInstances {
```

Comment:
this `extends JsonValueInstances` is not doing anything here anymore, since what you have in the emit is an instance of `ResponseToJsonType`